torch_xla/core/dynamo_bridge.py

```diff
@@ -417,11 +417,19 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
 
   # partition the model and exectue to collect inputs
   supported_ops = XlaOperatorSupport()
-  partitioner = CapabilityBasedPartitioner(xla_model, supported_ops)
+  partitioner = CapabilityBasedPartitioner(
+      xla_model, supported_ops, allows_single_node_partition=True)
   partitions = partitioner.propose_partitions()
   partitioned_graph = partitioner.fuse_partitions(partitions)
   InputCollector(partitioned_graph).run(*xla_args)
 
+  # TODO (@wonjoo) Add some comment
+  self_tensors = []
+  for name, buffer in xla_model.named_buffers():
+    if "self" in name:
+      self_tensors.append(buffer)
+  torch_xla._XLAC._xla_sync_multi(self_tensors, devices=[], wait=True)
```

Comment:
Need a way to explicitly materialize the tensor, hence the `_xla_sync_multi` call.

Reply:
why do we need this? Didn't we do `mark_step` on the entrance of the `extract graph`?